manifests/ingress/nginx-kube-lego/README.md

```diff
@@ -0,0 +1,49 @@
+# Ingress with HTTPS enable
+
+- Kube-lego for certificate retrieval from Let's Encrypt
+- nginx ingress-controller
+
+## Create nginx ingress-controller
+
+```
+kubectl apply -f nginx/ingress-controller.yaml
+```
+
+The nginx service uses NodePort to publish the service (for minikube only). Consider to change service type to LoadBalancer in public k8s cluster.
```

Comment:
Just change it to type=LoadBalancer by default.  It will work in minikube too (because LoadBalancer implies NodePort)